services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/pdf/HaittojenhallintasuunnitelmaPdfEncoder.kt

```diff
@@ -30,7 +35,18 @@ object HaittojenhallintasuunnitelmaPdfEncoder {
         document.subtitle(data.name)
 
         document.mapHeader("Alueiden sijainti", locationIcon)
-        document.placeholderImage()
+        // The image is better quality, if it's rendered at a higher resolution and then scaled down
```

Comment:
Would it be better to have this image generation in its own function?

Reply:
I extracted the image handling to a `Document.map()` function. I didn't add the spacer to this function yet. I want to see if I can use the same spacer for the other maps.